src/Profile/ProfileKeys.cpp

```diff
@@ -96,6 +96,7 @@ const char LanguageFile[] = "LanguageFile"; // pL
 const char InputFile[] = "InputFile"; // pL
 const char PilotName[] = "PilotName";
 const char CoPilotName[] = "CoPilotName";
+const char PilotWeGlideId[] = "PilotWeGlideId";
```

Comment:
I'd rather name it `WeGlidePilotId`, because there may be other WeGlide specific settings in the future, and having a common prefix is a good thing.